app/routers/users.py

```
@@ -0,0 +1,64 @@
+    return
+
+@router.delete("/me", summary="회원 삭제", description="계정을 삭제합니다.", tags=["users"])
+async def delete_user(request: UserLoginRequest):
+    return
+
+@router.get("/me", summary="내 프로필 조회", description="로그인한 사용자 본인 정보를 조회합니다", tags=["users"])
+async def get_me(request: UserLoginRequest):
+    return
+
+@router.get("/{user_id}", summary="특정 회원 조회", description="다른 사용자의 공개 프로필 조회", tags=["users"])
+async def get_user_by_id(request: UserLoginRequest, user_id: int):
```

Ruff (0.14.11)

Unused function argument: `request`

(ARG001)